rubygem-hammer_cli_foreman_docker/rubygem-hammer_cli_foreman_docker.spec

```diff
@@ -75,6 +75,7 @@ install -m 755 .%{gem_instdir}/config/foreman_docker.yml \
 %doc %{gem_instdir}/README.md
 %doc %{gem_instdir}/config
 %doc %{gem_instdir}/doc
+%doc %{gem_instdir}/LICENSE
```

Comment:
The licence must be shipped in the main package to accompany the code it licenses.